bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/WriteFlagTest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.client.api;
+
+import java.util.EnumSet;
+import static org.apache.bookkeeper.client.api.WriteFlag.DEFERRED_FORCE;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * Utit tests for WriteFlag
+ */
+public class WriteFlagTest {
+
+    @Test
+    public void testGetWriteFlagsDeferredForce() {
+        assertEquals(EnumSet.of(DEFERRED_FORCE), WriteFlag.getWriteFlags(1));
```

Comment:
give a meaningful mean for value `1`

Reply:
used DEFERRED_FORCE.getValue() or is it a dangerous shortcircuit ?